lib/msal-core/src/authority/B2cAuthority.ts

```diff
@@ -39,4 +31,13 @@ export class B2cAuthority extends AadAuthority {
 
         throw ClientConfigurationErrorMessage.unsupportedAuthorityValidation;
     }
+
+    /**
+     * Checks to see if the host is in a list of trusted hosts
+     * @param {string} The host to look up
+     */
+    public IsInTrustedHostList(host: string): boolean {
+        // Change this when we implement Known Authorities
+        return AADTrustedHostList[host.toLowerCase()];
+    }
```

Comment:
Do we need this for B2C? This may morph to the `knownAuthorities()` in the future; For now can we assume `validateAuthority: false` as pre requisite and remove this in `B2CAuthority.ts` all together?

Reply:
Could remove it. My thinking was to use this function to check against knownAuthorities, when implemented, to keep the structure in line with AAD. Can be discussed